### PR TITLE
unified admin user account

### PR DESCRIPTION
**Pull request by alanrenmsft**
azdata changes the environment variables for username and password. this PR updates the notebooks and wizard to reflect it. 

This will not be merged to master directly, there are other cu1 related changes, I will make the changes in the cu1 branch and merge to master later.

### CODE REVIEW

File: extensions/resource-deployment/src/ui/deployClusterWizard/pages/clusterSettingsPage.ts [hunk 1]
```diff
@@ -39,19 +39,20 @@ export class ClusterSettingsPage extends WizardPageBase<DeployClusterWizard> {
...
 					type: FieldType.Text,
-					label: localize('deployCluster.ControllerUsername', "Controller username"),
+					label: localize('deployCluster.AdminUsername', "Admin username"),
 					required: true,
 					variableName: VariableNames.AdminUserName_VariableName,
 					defaultValue: 'admin',
-					useCustomValidator: true
+					useCustomValidator: true,
+					description: localize('deployCluster.AdminUsernameDescription', "You can use this username to access controller and SQL Server, username for gateway is root.")
 				}, {
```
Comment (Charles-Gagnon) [inline extensions/resource-deployment/src/ui/deployClusterWizard/pages/clusterSettingsPage.ts]: I'd run this by PMs, but I think it reads better as "You can use this username to access the controller..."

Comment (alanrenmsft) [reply extensions/resource-deployment/src/ui/deployClusterWizard/pages/clusterSettingsPage.ts]: @dnethi what do you think?

Comment (chlafreniere) [reply extensions/resource-deployment/src/ui/deployClusterWizard/pages/clusterSettingsPage.ts]: I agre with Charles that putting a "the" makes it seem nicer.

In addition, I'd change the "," after "Server" to a ".", and capitalize the "U" in "Username"

File: extensions/resource-deployment/src/ui/deployClusterWizard/pages/clusterSettingsPage.ts [hunk 2]
```diff
@@ -39,19 +39,20 @@ export class ClusterSettingsPage extends WizardPageBase<DeployClusterWizard> {
...
 					type: FieldType.Password,
 					label: localize('deployCluster.AdminPassword', "Password"),
 					required: true,
 					variableName: VariableNames.AdminPassword_VariableName,
 					defaultValue: '',
 					useCustomValidator: true,
-					description: localize('deployCluster.AdminPasswordDescription', "You can also use this password to access SQL Server and gateway.")
+					description: localize('deployCluster.AdminPasswordDescription', "You can use this password to access controller, SQL Server and gateway.")
```
Comment (chlafreniere) [inline extensions/resource-deployment/src/ui/deployClusterWizard/pages/clusterSettingsPage.ts]: "the controller" sounds a bit more natural

**Pull request merged**